.github/workflows/release-admin-tools.yml

```diff
@@ -67,4 +67,4 @@ jobs:
           DST_REPO: temporalio
           LATEST: ${{ github.event.inputs.latest }}
           MAJOR: ${{ github.event.inputs.major }}
-        run: go run src/release_temporal/main.go
```

Comment:
`release_temporal` is a misnomer. What' is doing is to copy a Docker image from one place to another.